app/forms/provider_interface/recruit_with_pending_conditions_form.rb

```diff
@@ -0,0 +1,24 @@
+module ProviderInterface
+  class RecruitWithPendingConditionsForm
+    include ActiveModel::Model
+
+    attr_accessor :actor, :application_choice, :confirmation
+
+    validates :confirmation, presence: true
+    validates :confirmation, inclusion: { in: %w[yes no] }
+
+    def save
+      return false unless valid?
+
+      if confirmed?
```

Comment:
```suggestion     
      if valid? && confirmed?
```

Reply:
I think I still have to handle the _No_ radio button correctly (it's not the same end result as a validation fail).